src/components/Header/index.js

```diff
@@ -54,14 +46,17 @@ function HeaderIndex() {
               ))}
             </select>
           </div>
-          <div>
-            <FontAwesomeIcon icon={faUserCircle} />
+          <div className="header__user">
+            <AccountCircleIcon />
+            <label className="header__user__label">
+              {t("header.user.login")} / {t("header.user.signup")}
```

Comment:
You can also try creating another object at the top like this:
```js
const tPaths = {
  login: 'header.user.login',
  signup: 'header.user.signup'
}
```

Reply:
okay